app/views/layouts/decidim/admin/users.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for :secondary_nav do %>
+  <div class="secondary-nav">
+    <div class="secondary-nav__title">
+      <%= t ".title" %>
+    </div>
+    <ul>
+      <% if allowed_to? :read, :admin_user %>
+        <li <% if is_active_link?(decidim_admin.users_path) %> class="is-active" <% end %>>
+          <%= link_to t("menu.admins", scope: "decidim.admin"), decidim_admin.users_path %>
+        </li>
+      <% end %>
+      <% if current_organization.user_groups_enabled? && allowed_to?(:index, :user_group) %>
+        <li <% if is_active_link?(decidim_admin.user_groups_path) %> class="is-active" <% end %>>
+          <%= link_to t("menu.user_groups", scope: "decidim.admin"), decidim_admin.user_groups_path %>
+        </li>
+      <% end %>
+      <% if allowed_to? :index, :officialization %>
+        <li <% if is_active_link?(decidim_admin.officializations_path) %> class="is-active" <% end %>>
+          <%= link_to t("menu.participants", scope: "decidim.admin"), decidim_admin.officializations_path %>
+        </li>
+      <% end %>
+      <% if allowed_to? :index, :impersonatable_user %>
+        <li <% if is_active_link?(decidim_admin.impersonatable_users_path) %> class="is-active" <% end %>>
+          <%= link_to t("menu.impersonations", scope: "decidim.admin"), decidim_admin.impersonatable_users_path %>
+        </li>
+      <% end %>
+      <% if allowed_to? :index, :authorization_workflow %>
```

Comment:
I had to override this file because this line doesn't exist in the original file.

Reply:
oh.. what a pity, can you add your comment to the code please?